### DISCUSSION TITLE
vue defineComponent, alpinejs how to implement alternatively

### DISCUSSION THREAD

**Discussion by ericxin1982** [alpinejs/alpine#4114] [1. Help]
Hi,

I am looking for alpinejs way to achieve vue definecomponent, is it possible?

Thanks
Eric Xin

**Comment by ekwoka** [comment]
In what sense?

Like making a web component? Or?

AlpineJS does not do templating that would use components in that sense. That should be handled in your existing templating system.

Alpinejs has Alpine.data to handle defining and reusing data components.
